lib/watcher.ts

```diff
@@ -199,8 +164,28 @@ class Watcher extends EventEmitter {
         logger.debug('buildFailure');
         this.emit('buildFailure', err);
       }
-    );
-    return buildPromise;
+    ).finally(() => this._activeBuild = false);
```

Comment:
TODO: technically we support node 8 still, so this needs to use syntax `finally`